modules/travel_pay/app/services/travel_pay/doc_reader.rb

```diff
@@ -0,0 +1,71 @@
+require 'nokogiri'
+require 'zip'
+
```

Comment:
I did not actually call this anywhere in the code, so feel free to delete this file with reckless abandon if it doesn't fit the bill.